diffkemp/simpll/Result.h

```diff
@@ -79,6 +79,20 @@ struct FunctionInfo {
     }
 };
 
+/// Type for storing information about definition of an analysed
+// 'object' (eg. macro).
+struct Definition {
+    // Name of the 'object'.
+    std::string name;
+    // Line in the sourceFile on which is located definition of the 'object'.
+    int line;
```

Comment:
Could be `unsigned`, right?

Reply:
Thanks, makes more sense for line to be `unsigned`, I changed it (originally in `MacroDef` it was `int` - but the number that is saved in it is `unsigned` https://github.com/diffkemp/diffkemp/blob/e6cded9aefd2e51ada26c972f499fe08d442d789/diffkemp/simpll/SourceCodeUtils.cpp#L370).